### ISSUE TITLE
Design Intent - Design Feedback - Benefits Management Tools, Claim Status Tool, Improve Evidence Request/Submission Experience

### ISSUE THREAD

**Issue by allison0034** [department-of-veterans-affairs/va.gov-team#61470]
## VFS actions
- [x] Assign this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [x] Comment on this ticket:
  - [ ] If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - [ ] When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - [ ] When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] Close the ticket when all feedback has been addressed.

## Thoughts/questions

- These are looking great. I would love to see a mobile view and some more formal user flows.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- **Must**: Review all the alerts and featured content and make sure it matches the [guidelines](https://design.va.gov/components/alert#usage). Don't forget a [warning alert](https://design.va.gov/components/alert#warning-alert) should have a negative consequence. Read the content and decide wether or not it really needs to be an alert or featured content.
- <img width="422" alt="Screen Shot 2023-07-06 at 10 11 54 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/31019c7b-f182-4c35-a895-d7bc88f3e07f">
- **Must**: Per [our guidance](https://design.va.gov/components/alert#how-to-use-alerts), do not stack alerts
- **Must**: Remove the blue bottom border under any headings and follow the normal heading guidelines.
- <img width="352" alt="Screen Shot 2023-07-06 at 9 05 23 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/f4285494-fd4b-4410-a8dc-b619dfdbb9f0">
- **Must**: Talk to design system council about the [divider](https://github.com/orgs/department-of-veterans-affairs/projects/946/views/3?pane=issue&itemId=29995748). This recently came up as an item to add to the design system.
- **Must**: We do have established UI and pattern around [uploading files](https://design.va.gov/patterns/ask-users-for/files). Please review that and following the guidance. Note, some of the examples use the word "document" however content made the decision a couple of months ago to use "files" instead. We are working on updating the examples. If you make the change please send a screen shot to Allison : )
- **Must**: I am excited to see the lightweight card style! Accessibility worked on getting the border to be the correct contrast ratio and we have it added to our [design system](https://design.va.gov/components/card#default---white-with-border). Use the gray-medium for the border color. 
- **Must**: We currently only use gray for [tags](https://design.va.gov/components/tag), however other color options may be useful. Talk to design system council about using different colors for the tags and be sure to test around that. Move the status tag to the top of the card per [our guidance](https://design.va.gov/components/card#status-via-tag) and then the [status with indicator](https://design.va.gov/components/card#status-via-label-with-indicator) just below the header. I believe you said accessibility wanted these tags in a different spot and if so that is fine, please just comment on this ticket and let us know the final decision so we can update the design system. Remove the clock next to status and replace with green circle with a check to stay consistent with other statuses on VA.gov. Remove the check icon next to last updated as that is used for status.
- <img width="431" alt="Screen Shot 2023-07-06 at 9 34 49 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/09822ed2-99c3-4040-97bd-3594b2042a7c">
- **Must**: Remove the alert styling from "You don't have to do anything...". We have noticed alert fatigue on VA.gov and per [our guidance](https://design.va.gov/components/alert#success-alert) this alert style should be used to indicate success.
- <img width="493" alt="Screen Shot 2023-07-06 at 9 50 03 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/cee2c810-9888-476d-bf45-13bab001d848">
- **Must**: Follow the current UI and pattern for the [process list](https://design.va.gov/components/process-list). If you would like to explore using different colors/icons talk to design system council. Revaluate the guidance and using additional info component within the process list. If it does meet the guidance, use the [no border](https://design.va.gov/components/additional-info#no-border) since the process list is using the left column. Pay attention to the amount of content you are using and the guidance. Our process list guidance needs some updating, but I do not believe you should be using the expandable header in the process list, although talk to DSC and accessibility. 
- <img width="278" alt="Screen Shot 2023-07-06 at 10 23 24 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/f7ef275a-8a2c-46ae-a7b6-1e63dbcfe7fe">

- **Should**: I do not think using a [featured content block](https://design.va.gov/components/featured-content) on this page is necessary. If you use the proper heading structure this page will be easy to scan.
- <img width="494" alt="Screen Shot 2023-07-06 at 9 55 41 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/59b1a523-7e1a-47fc-8b7e-23adec7487fa">

**Comment by skylerschain** [comment]
Thanks for taking a look at this and all the great feedback. Just a couple questions: 

- In the second bullet "Do not stack alerts" is this referring to a specific part of the UI?
- Cool about the lightweight card style! I didn't know that had been created so happy to see there's an existing component for that.
- For removing the alert styling for "You don't have to do anything...": Point taken about overuse of alert styles and I completely agree. That said, we saw very strong positive response to this message in our research, multiple participants calling it out as the most useful part of the page. I'm worried if we reduce it to plain text we will lose the value of that strong reassurance message. I wonder if there is a way to retain the prominence and visual strength of that message without it being an alert.

**Comment by allison0034** [comment]
Hi @skylerschain!

- Yes, I noticed it on the screen below. And this is also a screen that needs to have the alerts be reevaluated. Do both of these really need to be alerts? The answer may be yes but just double check.
- <img width="313" alt="Screen Shot 2023-07-10 at 4 12 04 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/72398073/2e490194-d7a9-44c7-926a-9095b443178f">
- I think if you use the proper heading structure it would make the content clear and easy to scan. You could possibly use the [intro text styling](https://design.va.gov/foundation/typography#paragraphs) for the "You don't have anything to do right now" text. I would suggest user testing without the alert and seeing if that content is missed. Or you could remove the [featured content styling](https://design.va.gov/components/featured-content) from the content below it and put it on that text instead. I just need to confirm featured content is not for eligibility only anymore. You could also consider merging that content (what you need to do and what we're doing. Do they really need to be separate? What is the other content for those areas?

**Comment by skylerschain** [comment]
Got it, thanks @allison0034!

**Comment by allison0034** [comment]
@skylerschain you could also explore some experimental design and a way to feature content that is different than the blue box. We do have a need for that on VA.gov. Something that isn't as prominent as an alert but not just plan paragraph text. Gov.uk uses something called [inset text](https://design-system.service.gov.uk/components/inset-text/). They do warn that "Some users do not notice inset text if it’s used on complex pages or near to other visually prominent elements."  But you could explore some design options.

I think if you make normal headings and remove alert styling and featured content on both areas of content you might see a clearer path on what to do. Merging content may make sense then and having intro text may help. Also consider what info you have on the overview tab vs on this tab. Do you need both? Is it the same info just displayed differently?

You could even consider moving the content in that lightweight card at the top "What you've claimed". Maybe that content can be reworked into the content below? Don't feel like you have to stick with the layout that is currently there. If it doesn't make sense then you can change it.

**Comment by skylerschain** [comment]
Another question @allison0034 about the [status indicator](https://design.va.gov/components/card#status-via-label-with-indicator) you linked above: is the guidance to use the green check mark as the icon regardless of the object's status? For a closed claim it will say "Status: Closed" and it looks a bit off with the green check in that state. But if that's the guidance I'm happy to go with it. I wasn't sure if that scenario was considered.

// Also I'm happy to post in the design platform Slack if that is a better forum for these questions.

**Comment by allison0034** [comment]
@skylerschain not necessarily. You can use an icon that relates to the content. We are trying to align icon usage across VA so take a look at our [current icons](https://design.va.gov/foundation/icons). We are hoping to get statuses aligned as well so if you have suggests please let us know. The green circle with the check is most common for "complete" but we do not have a full audit of all the options.

**Comment by skylerschain** [comment]
#### Background on icons

A little more context on how we landed on this icon treatment. In the current CST, we are using icons to call out: 
1. When a Veteran has a new letter related to the claim 
2. When there is an item that needs the Veteran's attention (usually a document request)

<img width="640" alt="Screenshot 2023-04-10 at 2 33 22 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/ed3a99a4-8d3a-4f7e-8b6d-16c1ecefd814">

These designs predate our team, so the goal here was to clean up the icon story and apply it more consistently. If each line of content within the claim card is prefixed with an icon, it creates a nice visual alignment on the left and I think a sense of consistency and order on the card. The current icons feel a bit disjointed due to sizing and color discrepancies. 

#### Order of icons 
You also mentioned in the original feedback moving the "Status" icon up below the header. The reason for not doing that in the original proposal, was so that the alert items "You have a decision letter ready" and "An item needs your attention" could appear at the top to increase their visibility, given that this is new information we think the Veteran wants to know.

<img width="637" alt="Group 22 Copy 4" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/d23949b3-fe55-4dfd-acd4-a50ab41c69b3">

<img width="637" alt="Group 22 Copy 3" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/a44dfd7e-ea3b-4ce6-b5cd-31a6df40f78a">

#### Icon choices 

The icons in the original design proposal evolved as I explored the variations and those felt like they fit the best together. Removing the green and making all the icons monochrome felt like it leveled the hierarchy and tidied things up a bit. It's not clear to me why the status is more important than any other piece of information on the card (especially since with these claims, "Evidence gathering, review, and decision" is one big status and so it doesn't change very often). 

<img width="637" alt="Group 22 Copy 3" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/1c5c8548-432b-4b95-9152-b9cedb007b7c">

#### Other options

If the status icon definitely needs to be the green check, we can do that, I think we will then just need to figure out what the other icons need to be as it doesn't work as well with the other check mark below for "Last updated."
 
<img width="637" alt="Group 22 Copy 8" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/03bb600e-201a-4b14-9d2c-78e4361d30b8">

<img width="637" alt="Group 22 Copy 14" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/f6f105b3-0f56-41bb-8bca-c8ba151b20c6">

#### In conclusion

I actually don't think we'll have time for this at DSC this Friday because I have two other items to discuss :) But I would love to work together to figure this out and align on the icon story. Let me know if you have initial thoughts or want to set up a meeting to discuss.

**Comment by allison0034** [comment]
@skylerschain thanks for the info, it is helpful. Why don't you sign up for [design office hours](https://calendly.com/platform-office-hours/design-office-hours?back=1&month=2023-07) next Tuesday and I will have Matt and Brian attend and we can walk thru your ideas.

One problem I can see is that some of these icons you are using for Last updated are already used elsewhere on VA for different content. Do you really need an icon for last updated date? No icon might be something to consider.

We do not have a full audit but we have seen the green circle with the check and green check as the 2 most common things for a status being complete.

We definitely need help on trying to align statuses across VA. I think the struggle is trying to find an icon that has a direct relationship to the content (eg what icon means gathering evidence).

One more thing to consider... does every status need an icon especially if you have the status in the tag? We know from user research that those tags typically draw attention when scanning. We currently do not have any designs that have both the tag and a separate status with icon like in your designs that I am aware of. And I am not saying that is wrong, it may work. The tag may be a quick thing to scan and easily know what the status is for that item and then the other status may be for more details. Which is why I wonder if you even need icons with that detailed status? Might be something to explore.

So, I think this would be a good discussion. Let me know if that Tuesday time works. My hope is you can help us establish specific icons for statuses we can use across VA. : )

**Comment by skylerschain** [comment]
Yeah, I'll sign up for Tuesday! Looking forward to chatting more about all this.

**Comment by skylerschain** [comment]
A couple of updates on this work: 

- We met with the DSC and decided to hold off on introducing color to tags. I'm now using the current tags with gray background for in progress claims only.

- After meeting with the platform team at office hours, I've switched over to the new recommended lightweight card style. This seems to be the direction the design system as a whole is headed. This also helps draw attention to claims that require a user's attention, using the slim alert style as you can see above. 

- The rest of the "Must" and "Should" feedback should now be addressed, and we can review again at the Midpoint review, which I'll schedule as soon as I close out the other tickets. @allison0034 let me know if you have anything you'd like to discuss before then!

 
<img width="572" alt="Screenshot 2023-07-24 at 12 13 24 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/3f7c8835-30b3-480e-97d4-a9c1676a559b">
<img width="566" alt="Screenshot 2023-07-24 at 12 13 57 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/95102058/5291c008-c1c6-4f42-8104-3ce5a8203455">
